trusted_os/rpmb_fake.go

```diff
@@ -45,12 +46,16 @@ const (
 )
 
 type RPMB struct {
-	mem     [numSectors][sectorLength]byte
+	mem     [numSectors][]byte
 	counter uint32
 }
 
 func newRPMB(_ Card) (*RPMB, error) {
-	return &RPMB{}, nil
+	r := &RPMB{}
```

Comment:
I guess we'll do a follow up to write these out to MMC blocks so we can exercise things like version comparison and witness ID incrementing across reboots?

Reply:
Ugh, right, these need to be written to MMC to persist. Will follow up.